pkg/odo/cli/component/create.go

```diff
@@ -881,6 +881,101 @@ func (co *CreateOptions) Validate() (err error) {
 	return nil
 }
 
+func downloadGitProject(starterProject *devfilev1.StarterProject, starterToken, path string) error {
+
+	var projectSource devfilev1.GitLikeProjectSource
+	if starterProject.Git != nil {
+		projectSource = starterProject.Git.GitLikeProjectSource
+	} else {
+		projectSource = starterProject.Github.GitLikeProjectSource
+	}
+
+	remoteName, remoteUrl, revision, err := parsercommon.GetDefaultSource(projectSource)
+	if err != nil {
+		return errors.Wrapf(err, "unable to get default project source for starter project %s", starterProject.Name)
+	}
+
+	refName := plumbing.ReferenceName(revision)
```

Comment:
Some comment indicating `refName` is actually a branch. I know it's silly to ask but this might confuse a new contributor.

Reply:
added little details